### ISSUE TITLE
Disabling Telemetry still tries to connect to Mixpanel

### ISSUE THREAD

**Issue by halfdan** [linkedin/datahub#4299]
**Describe the bug**
I am on a network restricted Mac where I can't connect to Mixpanel. I have tried disabling telemetry via the CLI and can confirm telemetry is disabled according to `.datahub/telemetry-config.json`. Running any command however I immediately get the following error:

```
Traceback (most recent call last):
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/datahub/__main__.py", line 1, in <module>
    from datahub.entrypoints import main
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/datahub/entrypoints.py", line 11, in <module>
    from datahub.cli.delete_cli import delete
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/datahub/cli/delete_cli.py", line 21, in <module>
    from datahub.telemetry import telemetry
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/datahub/telemetry/telemetry.py", line 147, in <module>
    telemetry_instance = Telemetry()
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/datahub/telemetry/telemetry.py", line 46, in __init__
    mp.people_set(
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/mixpanel/__init__.py", line 238, in people_set
    return self.people_update({
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/mixpanel/__init__.py", line 392, in people_update
    self._consumer.send('people', json_dumps(record, cls=self._serializer))
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/mixpanel/__init__.py", line 594, in send
    self._write_request(self._endpoints[endpoint], json_message, api_key, api_secret)
  File "/Users/fbecker18/.asdf/installs/python/3.9.6/lib/python3.9/site-packages/mixpanel/__init__.py", line 623, in _write_request
    six.raise_from(MixpanelException(e), e)
  File "<string>", line 3, in raise_from
mixpanel.MixpanelException: HTTPSConnectionPool(host='api.mixpanel.com', port=443): Max retries exceeded with url: /engage (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1129)')))
```

Telemetry config is as follows:
```
{
  "client_id": "REMOVED",
  "enabled": false
}
```

**To Reproduce**
Fresh pip install, no additional steps needed.

**Expected behavior**
When telemetry is disabled I expect no outgoing telemetry requests.

**Desktop (please complete the following information):**
 - OS: Mac OS, 11.6
```
❯ pip freeze | grep datahub
acryl-datahub==0.8.27.1
```

**Comment by halfdan** [comment]
Looks like this was fixed in main (https://github.com/linkedin/datahub/commit/3d6db93aa67365b2029187299ac06195f72a7895) but hasn't been released yet. Closing issue and awaiting release.

**Comment by persistZzzZ** [comment]
> Looks like this was fixed in main ([3d6db93](https://github.com/linkedin/datahub/commit/3d6db93aa67365b2029187299ac06195f72a7895)) but hasn't been released yet. Closing issue and awaiting release.

Excuse me, when will it be released because I also encountered this problem
